components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.spec.ts

```diff
@@ -24,4 +23,62 @@ describe('ProjectsDropdownComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('#projectsArray', () => {
```

Comment:
Why the octothorp? Hmm... I see some tests use that... but most do not. 🤷‍♂ Personally I do not see that it adds value.

Reply:
In the Angular docs they used an Octothorpe to represent a function.   I thought this might be convention 🤷‍♂ I'll pull this out!